packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -51,48 +48,44 @@ export function SquadHeaderBar({
   const totalStepsCount = steps.length;
   const checklistTooltipText = `${completedStepsCount}/${totalStepsCount}`;
   const showJoinButton = squad.public && !squad.currentMember;
+  const firstItemClasses = 'order-5 tablet:order-1';
 
   return (
     <div
       {...props}
       className={classNames(
-        'flex h-fit w-full flex-row justify-center gap-4 tablet:w-auto',
+        'flex h-fit w-full flex-row flex-wrap justify-center gap-4 tablet:w-auto',
         className,
       )}
     >
-      <div className="relative">
-        {verifyPermission(squad, SourcePermissions.Invite) &&
-          !showJoinButton && (
-            <Button
-              variant={ButtonVariant.Secondary}
-              className={classNames(
-                tourIndex === TourScreenIndex.CopyInvitation &&
-                  'highlight-pulse',
-              )}
-              onClick={() => {
-                trackAndCopyLink();
-              }}
-              icon={<AddUserIcon />}
-              disabled={copying}
-            >
-              Invitation link
-            </Button>
+      {verifyPermission(squad, SourcePermissions.Invite) && !showJoinButton && (
+        <Button
+          variant={ButtonVariant.Secondary}
+          className={classNames(
+            firstItemClasses,
+            tourIndex === TourScreenIndex.CopyInvitation && 'highlight-pulse',
           )}
-      </div>
+          onClick={() => {
+            trackAndCopyLink();
+          }}
```

Comment:
```suggestion
          onClick={trackAndCopyLink}
```

Reply:
This function accepts an optional parameter that doesn't match native `onClick`.

Reply:
Ah ok no problem